cmd/bench.go

```diff
@@ -15,13 +14,11 @@ import (
 	"testing"
 
 	"github.com/olekukonko/tablewriter"
-	"github.com/spf13/cobra"
-
 	"github.com/open-policy-agent/opa/internal/presentation"
-
 	"github.com/open-policy-agent/opa/metrics"
 	"github.com/open-policy-agent/opa/rego"
 	"github.com/open-policy-agent/opa/util"
+	"github.com/spf13/cobra"
```

Comment:
Well I had decided to turn a blind eye, the linter never does that, fortunately 😄 https://github.com/open-policy-agent/opa/pull/3049/checks?check_run_id=1665309105#step:3:180